mod-source-record-manager-server/src/test/java/org/folio/rest/impl/metadataProvider/MetaDataProviderJobLogEntriesAPITest.java

```diff
@@ -314,7 +317,173 @@ public void shouldReturnLimitedCollectionOnGetWithLimitAndOffset(TestContext con
     }));
   }
 
-  private Future<JournalRecord> createJournalRecord(String jobExecutionId, String sourceId, String title, int recordOrder, JournalRecord.ActionType actionType,
+
+
+  @Test
+  public void shouldReturnEmptyDtoIfHasNoLogRecordsBySpecifiedJobIdAndRecordId() {
+    RestAssured.given()
+      .spec(spec)
+      .when()
+      .get(GET_JOB_EXECUTION_JOURNAL_RECORDS_PATH + "/" + UUID.randomUUID().toString() + "/records/" + UUID.randomUUID().toString())
+      .then()
+      .statusCode(HttpStatus.SC_OK);
```

Comment:
maybe it would better to return 404 status code instead of empty object when has no information for specified jobId and recordId